examples/examples-source/cookies_load.py

```diff
@@ -0,0 +1,33 @@
+import vrchatapi
+from http.cookiejar import Cookie
+
+from vrchatapi.api import authentication_api
+
+
+def make_cookie(name, value):
+    return Cookie(0, name, value,
+                  None, False,
+                  "api.vrchat.cloud", True, False,
+                  "/", False,
+                  False,
+                  173106866300,
+                  False,
+                  None,
+                  None, {})
+
+
+configuration = vrchatapi.Configuration(
+    username='username',
+    password='password',
+)
+
+with vrchatapi.ApiClient(configuration) as api_client:
+    api_client.user_agent = "MyProject/1.0 my@email.com"
```

Comment:
All our examples should use the same example agent, which VRChat blocks by default, forcing them to change it.

Reply:
Then Rust needs that changed.